feature/google/src/main/java/com/revenuecat/purchases/google/storeProductConversions.kt

```diff
@@ -60,9 +64,19 @@ fun List<ProductDetails>.toStoreProducts(): List<StoreProduct> {
             val offerDetailsForBasePlan = offerDetailsBySubPeriod[basePlanBillingPeriod] ?: emptyList()
 
             productDetails.toStoreProduct(offerDetailsForBasePlan).let {
+                if (it != null) {
```

Comment:
if we make both of these `?.let` then we don't need the null checks inside and can finish with `?: //log`